website/thaliawebsite/settings.py

```diff
@@ -0,0 +1,604 @@
+"""
+Settings module
+
+This file controls what settings are loaded.
+
+Using environment variables you can control the loading of various
+overrides.
+Django settings for thaliawebsite project.
+
+This file is loaded by __init__.py.
+
+Its settings may be overridden by files loaded after it.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/dev/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/dev/ref/settings/
+"""
+
+import logging
+
+import base64
+import json
+import os
+
+from django.core.management.commands import makemessages
+from django.utils import timezone
+from django.utils.translation import gettext_lazy as _
+
+logger = logging.getLogger(__name__)
+
+# Sentinel objects that are distinct from None
+_CRASH = object()
+_NOT_SET = object()
+
+
+class Misconfiguration(Exception):
+    pass
+
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.abspath(
+    os.path.join(os.path.dirname(os.path.abspath(__file__)), "", "..")
+)
+
+SOURCE_COMMIT = os.environ.get("SOURCE_COMMIT", "unknown")
+
+# Many of the settings are dependent on the environment we're running in.
+# The default environment is local, because the environment is set to production in wsgi.py
+DJANGO_ENV = os.environ.get("DJANGO_ENV", "local")
+_environments = ["production", "staging", "testing", "local"]
+if DJANGO_ENV not in _environments:
+    raise Misconfiguration(f"Set DJANGO_ENV to one of: {', '.join(_environments)}")
+
+
+def setting(*, local, production, staging=_NOT_SET, testing=_NOT_SET):
+    if DJANGO_ENV == "local" or testing is _NOT_SET:
+        return local
+    if DJANGO_ENV == "testing":
+        return testing
+    if DJANGO_ENV == "production" or staging is _NOT_SET:
+        return production
+    if DJANGO_ENV == "staging":
+        return staging
+    raise Misconfiguration(f"Set DJANGO_ENV to one of: {', '.join(_environments)}")
+
+
+def from_env(
```

Comment:
It would be nice to make it a bit clearer what happens when staging or testing is not set